src/base/emu-i386/simx86/cpu-emu.c

```diff
@@ -606,7 +606,7 @@ static void Scp2Cpu (sigcontext_t *scp)
   TheCPU.ds = _ds;
   TheCPU.es = _es;
 
-  TheCPU.scp_err = _err;
+  TheCPU.scp_err = 0;
```

Comment:
Hmm, perhaps there are more "output-only" params?
_cr2 for example?

Reply:
yes, _cr2 is another one but it's probably not a big deal. In any case I can submit a "TheCPU.cr2" cleanup patch in a different PR, since it would be nice to have that relative to mem_base.

Reply:
OK, thanks!
I guess there are more: cr2, trapno and all that.

Reply:
trapno is already one-way, so it's just cr2 in Scp2Cpu.